lib/matplotlib/axes/_secondary_axes.py

```diff
@@ -0,0 +1,466 @@
+import collections
+import numpy as np
+import numbers
+
+import warnings
+
+import matplotlib.docstring as docstring
+import matplotlib.ticker as mticker
+import matplotlib.transforms as mtransforms
+import matplotlib.scale as mscale
+import matplotlib.cbook as cbook
+
+from matplotlib.axes._base import _AxesBase
+
+from matplotlib.ticker import (
+    AutoLocator,
+    FixedLocator,
+    NullLocator,
+    NullFormatter,
+    FuncFormatter,
+    ScalarFormatter,
+    AutoMinorLocator,
+    LogLocator,
+    LogFormatterSciNotation
```

Comment:
You can sort the imports alphabetically, or put the locators on one side and the formatters on the other, I'm trying not to be overly dogmatic but here the order looks pretty random :)

Reply:
... its chronological...